### DISCUSSION TITLE
Is there a KVM image for test?

### DISCUSSION THREAD

**Discussion by qzhuyan** [amazonlinux/amazon-linux-2022#100] [General]
hi,

Amazon linux 2 has kvm image as in this link
https://cdn.amazonlinux.com/os-images/2.0.20220207.1/kvm/

Will amazon-linux-2022 have the kvm img as well?

**Comment by bplessis-swi** [comment]
By the way i found this in the following FAQ:

> Q: Can I use AL2022 outside of AWS?
> 
> A: AL2022 images can be used outside of AWS, however, these images are not covered by [AWS Support Plans](https://aws.amazon.com/premiumsupport/plans/) when used outside of AWS. When Amazon Linux 2022 becomes Generally Available, it will be provided as a virtual machine image for on-premises use, enabling you to easily develop, test, and certify applications from a local development environment. This option is not available during the preview.
> 

 https://aws.amazon.com/linux/amazon-linux-2022/faqs/?nc1=h_ls

**Comment by Xe** [comment]
Is this still planned?
